src/Host.cpp

```diff
@@ -449,6 +450,10 @@ Host::Host(int port, const QString& hostname, const QString& login, const QStrin
     if (mudlet::scmIsPublicTestVersion) {
         thankForUsingPTB();
     }
+
+    connect(&mTelnet, &cTelnet::signal_disconnected, this, [this](){ purgeTimer.start(1min); });
```

Comment:
Why does the slot called here and in the next line have to be done as a lambda? 😕

Reply:
Well, it's a simple start/stop here, and the lambda is the part where the explanation of the "why" of this system is, it's a bit bigger - doesn't feel right for the constructor, you know?